TheraLang.Web/Validators/MemberFeeViewModelValidators.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using TheraLang.Web.ViewModels;
+
+namespace TheraLang.Web.Validators
+{
+    public class MemberFeeViewModelValidators: AbstractValidator<MemberFeeViewModel>
+    {
+        public MemberFeeViewModelValidators()
+        {
+            RuleFor(x => x.FeeDate).NotNull().NotEmpty();
```

Comment:
I think if date is not null it can't be empty, so NotEmpty is useless here

Reply:
Done)